### PR TITLE
Additional resources

### LINKED ISSUE #126: Improve additional resources sections
Add a few sentences to give context to each additional resource we share. This means going just beyond annotating them (we should also do that), but also explain what topic they should focus on next. Rebut: We will not add a where to go from here chapter, asked by reviewer ?, but will instead do better job at the end of each chapter, on a topic-by-topic basis.
MINOR

### PR DESCRIPTION
**Pull request by trevorcampbell**
This PR resolves #126. I've expanded each additional resources section and made it clear what the scope of each resource is / why they would seek it out. I decided to include one at the end of classification and regression each (not every chapter, just the last in each pair) as I think it makes more sense to only provide resources after the topic has been fully covered.

### CODE REVIEW

**Comment by leem44** [comment]
@trevorcampbell Looks good! I made some super minor edits to the text itself! Will leave this PR open in case @ttimbers wants to have a look!

File: 02-wrangling.Rmd [hunk 1]
```diff
@@ -543,6 +543,6 @@ The `map_*` functions are generally quite useful for solving problems involving
...
+- The [`dplyr` page on the tidyverse website](https://dplyr.tidyverse.org/) is where you should look if you want to learn more about the functions in this chapter, the full set of arguments you can use, and other related functions. The site also provides a very nice cheat sheet that summarizes many of the data wrangling functions from this chapter.
+- [R for Data Science](https://r4ds.had.co.nz/) has a few chapters related to data wrangling that goes into more depth than this book. For example, the [tidy data](https://r4ds.had.co.nz/tidy-data.html) chapter covers the concept of tidy data, missing values, `pivot_longer`/`pivot_wider`, and additional wrangling functions (like `unite` and `separate`); the [data transformation](https://r4ds.had.co.nz/transform.html) chapter covers `select`, `filter`, `arrange`, `mutate`, and `summarize`; and the [`map_*` functions](https://r4ds.had.co.nz/iteration.html#the-map-functions) chapter provides more about the `map_*` functions.
```
Comment (trevorcampbell) [inline 02-wrangling.Rmd:547]: > has a few chapters related to data wrangling that goes into

this should be "go" -- "a few chapters goes" is wrong

File: 10-inference.Rmd [hunk 2]
```diff
@@ -880,5 +880,5 @@ This chapter is only the beginning of the journey into statistical inference. We
...
+- Chapters 4 to 7 of [OpenIntro Statistics - Fourth Edition](https://www.openintro.org/) provide a good next step after Modern Dive. Although it is still certainly an introductory text, things get a bit more mathematical here. Depending on your background, you may actually want to start going through Chapters 1 to 3 first, where you will learn some fundamental basic concepts in probability theory. Although it may seem like a diversion, probability theory is *the language of statistics*; if you have a solid grasp of probability, more advanced statistics will come naturally to you!
```
Comment (trevorcampbell) [inline 10-inference.Rmd:884]: > "some fundamental basic concepts"

This should just be "some fundamental concepts"

Comment (leem44) [reply 10-inference.Rmd:884]: Oh yes -- missed that! I agree!

File: 02-wrangling.Rmd (hunk 1 above)
Comment (leem44) [reply 02-wrangling.Rmd:547]: Oh yup! please do change!

File: 02-wrangling.Rmd [hunk 3]
```diff
@@ -543,8 +543,6 @@ The `map_*` functions are generally quite useful for solving problems involving
...
-
-- [Data transformation](https://r4ds.had.co.nz/transform.html)
-- [Tidy data](https://r4ds.had.co.nz/tidy-data.html)
-- [The `map_*` functions](https://r4ds.had.co.nz/iteration.html#the-map-functions)
+- The [`dplyr` page on the tidyverse website](https://dplyr.tidyverse.org/) is where you should look if you want to learn more about the functions in this chapter, the full set of arguments you can use, and other related functions. The site also provides a very nice cheat sheet that summarizes many of the data wrangling functions from this chapter.
+- [R for Data Science](https://r4ds.had.co.nz/) has a few chapters related to data wrangling that go into more depth than this book. For example, the [tidy data](https://r4ds.had.co.nz/tidy-data.html) chapter covers the concept of tidy data, missing values, `pivot_longer`/`pivot_wider`, and additional wrangling functions (like `unite` and `separate`); the [data transformation](https://r4ds.had.co.nz/transform.html) chapter covers `select`, `filter`, `arrange`, `mutate`, and `summarize`; and the [`map_*` functions](https://r4ds.had.co.nz/iteration.html#the-map-functions) chapter provides more about the `map_*` functions.
```
Comment (trevorcampbell) [inline 02-wrangling.Rmd:547]: @leem44 per our slack convo, look here for possible mistakes I made in discussing extra funcs!

**Pull request merged**